### PR TITLE
Add individual collect payment actions

### PR DESCRIPTION
**Pull request by ericwindham**
Changes proposed in this pull request:
- This allows payment gateways to create their own actions

@wpninjas/developers

### CODE REVIEW

**Comment by klhall1987** [comment]
Found a couple of problems, on my form that I created before switching to this branch I couldn't get the checkout modal to pop up. 

I'm also getting a fatal error on the Stripe action no the front end.) Fatal error: Call to a member function get_setting() on array in C:\wamp64\www\wordpress\wp-content\plugins\ninja-forms-stripe\includes\PaymentGateway.php on line 692

File: includes/Actions/CollectPayment.php [hunk 1]
```diff
@@ -32,15 +32,30 @@
 
     /**
      * Constructor
+     *
+     * @param string $cp_nice_name
+     * @param string $cp_name
      */
-    public function __construct()
+    public function __construct( $cp_nice_name = 'Collect Payment',
```
Comment (Much2tall) [inline includes/Actions/CollectPayment.php]: $cp_nice_name = __( 'Collect Payment', 'ninja-forms' ),

File: includes/Actions/CollectPayment.php [hunk 2]
```diff
@@ -32,15 +32,30 @@
...
+	    $cp_name = 'collectpayment' )
     {
         parent::__construct();
 
-        $this->_nicename = __( 'Collect Payment', 'ninja-forms' );
+        // Set the nice name to what we passed in. 'Collect Payment' is default
+        $this->_nicename = __( $cp_nice_name, 'ninja-forms' );
```
Comment (Much2tall) [inline includes/Actions/CollectPayment.php]: $this->_nicename = $cp_nice_name;

**Comment by Much2tall** [comment]
@klhall1987 I was unable to reproduce the problems you were experiencing with this branch.

**Pull request merged**